docs/modules/ROOT/pages/sentinel.adoc

```diff
@@ -653,6 +653,102 @@ exports.handler = async function(params) {
 }
 ----
 
+[[customizing-notification-messages]]
+== Customizing Notification Messages
+
+You can optionally modify the notification message content and formatting using the checkbox below the notification channel selector.
+
+=== Message Syntax
+
+Custom notifications support a limited set of markdown syntax:
+
+* Bold
+* Italic
+* Links
```

Comment:
Let's add in place examples. Nothing too fancy, something like:

* Bold (`**this text is bold**`)
* Italic (`*this text is italicized*`)

etc